tools/regtesting/timings.py

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# author: Ole Schuett
+
+import sys
+import math
+
+#===============================================================================
+def main():
+    if(len(sys.argv) != 2):
+        print("Usage: timings.py <timings.txt>")
+        sys.exit(1)
+
+    fn = sys.argv[1]
+    timings = sorted([float(line.split()[0]) for line in open(fn).readlines()])
+
+    print('Plot: name="timings", title="Timings Distribution", ylabel="time [s]"')
```

Comment:
Please use

```python
from __future__ import print_function
```

to make sure `print()` is actually a function in Python 2.

Reply:
It does actually work, but for the wrong reason. Thanks!